cypress/integration/pages/password-recovery/send_email_spec.page.js

```diff
@@ -0,0 +1,36 @@
+describe('Password recovery flow', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.findByText('Esqueceu sua senha?').click();
+  });
+
+  it('successfully sends an e-mail to reset the password', () => {
+    cy.findByRole('textbox', { name: /endereço de email/i }).type(
+      'admin@example.com'
+    );
+
+    cy.findByRole('button', { name: /recuperar senha/i }).click();
+
+    const toastClass = '.Toastify__toast-body';
+    cy.get(toastClass)
+      .should('be.visible')
+      .should('have.text', 'E-mail de recuperação enviado com sucesso!');
+  });
+
+  it('fails to send an e-mail to a non-registered user', () => {
```

Comment:
How does it fail? I think you should be mocking the network request to simulate an error.